src/main/java/server/Server.java

```diff
@@ -0,0 +1,134 @@
+package server;
+
+
+import java.io.*;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Server {
+
+    private static final int PORT = 8080;
+    private static final String EXIT_COMMAND = "-exit";
+    private static final String FILE_COMMAND = "-file";
+
+    private ServerSocket serverSocket;
+    private List<ClientHandler> clients;
+
+    public Server() {
+        clients = new ArrayList<>();
+    }
+
+    public void start() {
+        try {
+            serverSocket = new ServerSocket(PORT);
+            System.out.println("Server started on port " + PORT);
+
+            while (true) {
+                Socket clientSocket = serverSocket.accept();
+                ClientHandler clientHandler = new ClientHandler(clientSocket);
+                clients.add(clientHandler);
+                clientHandler.start();
+                broadcastMessage("[SERVER] " + clientHandler.getClientName() + " connected successfully.");
+            }
+        } catch (IOException e) {
+            Logger logger = Logger.getLogger(Server.class.getName());
+            logger.log(Level.SEVERE, "Error occurred while starting the server.", e);
+        }
+    }
+
+    public void stop() {
+        try {
+            serverSocket.close();
+            for (ClientHandler client : clients) {
+                client.disconnect();
+            }
+        } catch (IOException e) {
+            Logger logger = Logger.getLogger(Server.class.getName());
```

Comment:
would be better to get the logger on the class level and then reuse it in methods